templates/account/partials/site.php

```diff
@@ -174,7 +174,7 @@ class="dashicons dashicons-arrow-right-alt2"></i>
                 <?php $row_index ++ ?>
                 <!--/ Blog ID -->
 
-                <?php if ( $is_registered && $install->user_id != $main_user->id ) : ?>
+                <?php if ( $install->user_id != $main_user->id ) : ?>

```

Comment:
@fajardoleo don't we want to check `is_object( $VARS['user'] )` as the replacement for `$is_registered`?

Reply:
@vovafeldman The `$is_registered` part was removed as the modified line is already being wrapped with `if ( $is_registered )...`. With regard to `$VARS['user']`, the "Account" page is loaded only when it's an object.